scalafix-core/shared/src/main/scala/scalafix/internal/config/ScalafixMetaconfigReaders.scala

```diff
@@ -25,6 +28,7 @@ import metaconfig.Configured.Ok
 import scalafix.internal.config.MetaconfigParser.{parser => hoconParser}
 import scalafix.internal.rule.ConfigRule
 import scalafix.patch.TreePatch
+import org.scalameta.logger
```

Comment:
Ah the good old extraneous `org.scalameta.logger` :)

Reply:
Yes, I should make it a top-level package like `pprint` so I don't have to import anything 😅